app/src/main/res/values/strings.xml

```diff
@@ -22,8 +22,10 @@
     <string name="complete_the_recipe">Complete the recipe above to see the output</string>
     <string name="enter_dicekey_by_hand">Enter the DiceKey by Hand</string>
```

Comment:
Looking at this and wondering why Hand is capitalized.

Reply:
Fixed